Makefile

```diff
@@ -13,7 +13,7 @@ lint:
 	yapf -i -r .
 
 test: init-test-db
-	python -m pytest .
+	python3 -m pytest .
```

Comment:
This shouldn't be necessary assuming your virtualenv is running.
Can you explain why you put this?

Reply:
I guess my venv isn't working right for whatever reason.

Reply:
I'll try messing around with the venv folder and see if I can remove that line.

Reply:
Works now.